litmus-portal/frontend/src/pages/MonitoringDashboardPage/index.tsx

```diff
@@ -696,74 +719,112 @@ const DashboardPage: React.FC = () => {
-            >
-              {selectedDashboardInformation.dashboardListForAgent.map(
-                (data: ListDashboardResponse) => {
-                  return (
-                    <MenuItem
-                      key={`${data.db_id}-monitoringDashboard`}
-                      value={data.db_id}
-                      onClick={() => {
-                        setSelectedDashboardInformation({
-                          ...selectedDashboardInformation,
-                          id: data.db_id,
-                          name: data.db_name,
-                          type: data.db_type,
-                        });
-                        dataSource.selectDataSource({
-                          selectedDataSourceURL: '',
-                          selectedDataSourceID: '',
-                          selectedDataSourceName: '',
-                        });
-                        setRefreshRate(0);
-                        setAnchorEl(null);
-                      }}
-                      className={
-                        data.db_id === selectedDashboardInformation.id
-                          ? classes.menuItemSelected
-                          : classes.menuItem
-                      }
-                    >
-                      <div className={classes.expDiv}>
-                        <Typography
-                          data-cy="switchDashboard"
-                          className={`${classes.btnText} ${classes.italic}`}
-                          variant="h5"
-                        >
-                          {data.db_name}
-                        </Typography>
-                      </div>
-                    </MenuItem>
-                  );
+
+            <TopNavButtons
+              isInfoToggledState={isInfoOpen}
+              switchIsInfoToggled={(toggleState: Boolean) => {
+                setIsInfoOpen(toggleState);
+              }}
+            />
+          </div>
+          {isInfoOpen && (
+            <div className={classes.infoDiv}>
+              <InfoDropdown
+                dashboardConfigurationDetails={{
+                  name: selectedDashboardInformation.name,
+                  type: selectedDashboardInformation.type,
+                  dataSourceName: selectedDataSource.selectedDataSourceName,
+                  dataSourceURL: selectedDataSource.selectedDataSourceURL,
+                  agent: selectedDashboardInformation.agentName,
+                }}
+                metricsToBeShown={
+                  selectedDashboardInformation.panelNameAndIDList
                 }
-              )}
-            </Menu>
-          </Typography>
+                applicationsToBeShown={[]}
+                postPanelSelectionRoutine={(selectedPanelList: string[]) => {
+                  setSelectedPanels(selectedPanelList);
+                }}
+                postApplicationSelectionRoutine={(
+                  selectedApplicationList: string[]
+                ) => {}}
```

Comment:
any reason why this is an empty function?

Reply:
yes we have targeted this for beta4, this is the initial part as its under dev feature @arkajyotiMukherjee